### ISSUE TITLE
Struktur einer Seite

### ISSUE THREAD

**Issue by philsupertramp** [philsupertramp/wiki#17]
Offene Discussion:

@joshinils
Ich bin gerade an einem Punkt, an dem ich nicht weiß welche Elemente für eine "Seite" notwendig sind, meine aktuelle Liste beinhaltet keine Tags:
- Titel [Text Feld]
- Slug (für URL zuständig, z.B. `/wik/Datenstrukturen-und-Algorithmen`) [Text Feld]
- Inhalt [Text Feld]
- Autor [Verlinkung zum Nutzer]
- Erstellungszeitpunkt [Datetime Feld]
- Bearbeiter [Verlinkung zum Nutzer]
- letzter Bearbeitungszeitpunkt [Datetime Feld]

Tags sind, nach #6 mMn unnötig, was denkst du?

**Comment by joshinils** [comment]
wenn man tags mittels text-templates umsetzen kann... eigentlich schon.

also ich meine, dass man halt ein template auf die seite setzt, das diese dann auf einer anderen erscheinen lässt in einer liste.
das template ufngiert dann halt als eintrag des tags und ist teil des inhalts.

wie man das dann umsetzen kann... ka.
dafür kenn ich mich mit den templates zu wenig aus.

**Comment by joshinils** [comment]
eine seite würde noch die vorherigen versionen beinhalten, oder wie soll das struktuiriert sein?
da hab ich keine ideen, da ich mich halt mit der ganzen geschichte hier nicht so auskenne wie du.

ansonsten wäre das halt so sortiert, dass eine seite nur auf eine version verweist, die dann die aktuellste ist.
und dann hat halt die version die ganzen daten wie titel, inhalt, autor, etc. nur ohne slug

**Comment by joshinils** [comment]
und wenn die seite separat von den versionen ist, kann auf die seite selber auch ein verweis ob die seite existiert, oder ob sie "gelöscht" ist (und halt nur die vorherigen versionen sichtbar sind über die historie)

**Comment by philsupertramp** [comment]
> eine seite würde noch die vorherigen versionen beinhalten, oder wie soll das struktuiriert sein?
> da hab ich keine ideen, da ich mich halt mit der ganzen geschichte hier nicht so auskenne wie du.
> 
> ansonsten wäre das halt so sortiert, dass eine seite nur auf eine version verweist, die dann die aktuellste ist.
> und dann hat halt die version die ganzen daten wie titel, inhalt, autor, etc. nur ohne slug

Ja, dafür dachte ich mir kann man entweder ne Kopie des Eintrags zu Zeitpunkt X erstellen, per default gibt man die aktuellste Version aus und wenn jemand möchte könne er eine vorherige Version betrachten. Genaueres können wir aber in #7 diskutieren.
